docs/changes/io.fits/16550.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fix bug for integer compressed images with blanks.
```

Comment:
```suggestion
Fix a spurious exception when reading integer compressed images with blanks.
```

Reply:
It is in fact a bug, but I'll accept this anyway!

Reply:
indeed it is, but it'll be marked as such anyway in the final changelog since the file name contains 'bugfix' :)